### ISSUE TITLE
Feature Request: Violin Plot

### ISSUE THREAD

**Issue by PANhuihuihuihui** [apexcharts/apexcharts.js#3957]
## Summary
Add [Violin Plot](https://mode.com/blog/violin-plot-examples/) or combine the Violin plot in the Box plot.
<img width="622" alt="image" src="https://github.com/apexcharts/apexcharts.js/assets/47149748/03189dd7-ee37-4f9a-9af9-5a7b3dde4be7">

## Intended Use Case

Give more information about the distribution of data.

**Comment by PANhuihuihuihui** [comment]
upup

**Comment by Masstronaut** [comment]
Yes violin charts are awesome! I'd love to see it support asymmetric violins as well, so that it can be used to compare the frequency distribution of 2 related values. For example, up/down votes over time where time is a shared axis and one side of the violin is up votes, the other is down votes.

**Comment by Masstronaut** [comment]
I was able to approximate a violin chart using an area chart by doing the following:
- map one side of my data set into negative values
- render equal range on the chart above and below zero to keep the "violin" centered
- set the plot to fill to origin:

```js
plotOptions: {
	area: {
		fillTo: 'origin'
	}
},
```

It still doesn't have the box & whisker / quartile information in the middle, which is not an issue for my use case.

**Comment by beldogueto** [comment]
Hi everyone! I'm bumping this post because it would be amazing to have violin plots in ApexCharts! Violin plots are a must for certain types of data visualization and would be a fantastic addition to the library
